### PR TITLE
Add ros_compatibility module

### PR DESCRIPTION
**Pull request by arkadiy-telegin**

### CODE REVIEW

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        def spin(self):
+            rclpy.spin(self)
+
+        def shutdown(self):
+            rclpy.shutdown()
+
+else:
+    raise NotImplementedError("Make sure you have valid " +
+                              "ROS_VERSION env variable");
```
Comment (FabianSchurig) [inline ros_compatibility/src/ros_compatibility/ros_compatible_node.py]: Unnecessary semicolon

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+if ROS_VERSION == 1:
+    import rospy
+
+
+    class QoSProfile():
+        def __init__(self, **kwargs):
+            self.depth = kwargs['depth']
+
+
+    class CompatibleNode(object):
```
Comment (FabianSchurig) [inline ros_compatibility/src/ros_compatibility/ros_compatible_node.py:14]: Could you add some docstring please

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import os
+
+ROS_VERSION = int(os.environ['ROS_VERSION'])
```
Comment (FabianSchurig) [inline ros_compatibility/src/ros_compatibility/ros_compatible_node.py]: This might already throw a key error if there is no `'ROS_VERSION'`

Comment (FabianSchurig) [reply ros_compatibility/src/ros_compatibility/ros_compatible_node.py]: Do we need the cast?

**Review by FabianSchurig — APPROVED** [review]

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py (hunk 3 above)
Comment (FabianSchurig) [reply ros_compatibility/src/ros_compatibility/ros_compatible_node.py]: use `.get()` instead

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        def shutdown(self):
+            rclpy.shutdown()
+
+else:
+    raise NotImplementedError("Make sure you have valid " +
+                              "ROS_VERSION env variable");
+
+
+def main():
+    print('This is a ros1 - ros2 compatibility module.',
```
Comment (dennisrmaier) [inline ros_compatibility/src/ros_compatibility/ros_compatible_node.py:90]: use "..." for strings in Python

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+          "It's not meant to be executed, but rather imported")
+
+
+if __name__ == '__main__':
```
Comment (dennisrmaier) [inline ros_compatibility/src/ros_compatibility/ros_compatible_node.py:94]: same here

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py (hunk 3 above)
Comment (FabianSchurig) [reply ros_compatibility/src/ros_compatibility/ros_compatible_node.py]: 5f914ea

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py (hunk 1 above)
Comment (FabianSchurig) [reply ros_compatibility/src/ros_compatibility/ros_compatible_node.py]: f2ff458

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py (hunk 5 above)
Comment (FabianSchurig) [reply ros_compatibility/src/ros_compatibility/ros_compatible_node.py:94]: 72dfc2e

File: ros_compatibility/src/ros_compatibility/ros_compatible_node.py (hunk 4 above)
Comment (FabianSchurig) [reply ros_compatibility/src/ros_compatibility/ros_compatible_node.py:90]: 72dfc2e

**Pull request merged**